lib/guisso.ex

```diff
@@ -88,7 +88,7 @@ defmodule Guisso do
   end
 
   defp redirect_uri(conn) do
-    Coherence.ControllerHelpers.router_helpers.session_url(conn, :oauth_callback)
+    Coherence.Controller.router_helpers.session_url(conn, :oauth_callback)
```

Comment:
This is the actual bit that needed to be changed.

The rest is deps upgrade & move from CircleCI to GHA.